Automattic/wp-calypso
# Staging sites: Add feature to plans

Related to https://github.com/Automattic/dotcom-forge/issues/1741

## Proposed Changes

This diff adds the appropriate changes for the new staging site feature as described [here](https://docs.google.com/spreadsheets/d/1cPh9HIVfTwUR--H05ncU4LnTuIKVD8kggg29MBDzu6U/edit#gid=0) to the plan pages

## Testing Instructions

- Visit `/plans/<your site>`
- Make sure that you see the following changes:

| **Before** | **After** |
| :---:         |    :---:       |
|![plans_li_before1](https://user-images.githubusercontent.com/497103/236384643-eb2d1fb8-9f85-4e4b-ab57-6cbaced22e26.png) | ![plans_li_after1](https://user-images.githubusercontent.com/497103/236384659-20407bce-5ea0-4617-9495-26f4f8087306.png)|
|![plans_li_before2](https://user-images.githubusercontent.com/497103/236384689-68e16fd0-52bc-4a72-b146-a5f5160eb2b5.png)|![plans_li_after2](https://user-images.githubusercontent.com/497103/236384711-ac01b206-e239-46ee-b35a-2003852eba53.png)|

UPDATE:
The wording on staging site tooltip has changed from `plural` to `singular`.




*

## Pre-merge Checklist
